app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
According to [Python docs for exceptions](https://docs.python.org/3/library/exceptions.html), you shouldn't inherit from `BaseException`
This class is used for system exceptions, like `KeyboardInterrupt`, `SystemExit` and base class `Exception`
Main reason to stop writing this, is that catching errors with bare `except:`, that are used in most of frameworks, would no catch exceptions, inherited from `BaseException`

```suggestion
class NotWearingMaskError(Exception):
```